src/app/screens/BoardsList/Components/AddBoardModal/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import AddBoardModal from './layout';
+import './styles.css';
+
+class AddBoardModalContainer extends Component {
+  state = { visible: false };
+
+  openModal = () => {
+    this.setState({
+      visible: true
+    });
+  };
+
+  closeModal = () => {
+    this.setState({
+      visible: false
+    });
+  };
```

Comment:
Same here 😇

Reply:
I [don't mind](https://www.youtube.com/watch?v=ussCHoQttyQ) it this way.